src/applications/mhv/medications/util/constants.js

```diff
@@ -72,11 +72,18 @@ export const medicationsUrls = {
   MEDICATIONS_LOGIN: '/my-health/medications?next=loginModal&oauth=true',
 };
 
-export const refillStatus = [
+export const refillStatusForRefillsReft = [
```

Comment:
if we get rid of the refillStatus can we possibly get rid of these constants? what do you think?